src/dygraph-layout.js

```diff
@@ -269,6 +271,14 @@ DygraphLayout.prototype._evaluateLineCharts = function() {
         }
       }
       point.y = DygraphLayout.calcYNormal_(axis, yval, logscale);
+      outOfYBounds += ((point.y<0)||(point.y>1));
+    }
+
+    if (outOfXBounds > 0) {
```

Comment:
I wonder whether this even makes sense for the x axis… or rather, when zooming in, some points are always going to be out of bounds, also for the y axis. Maybe the checks should be conditional on not having zoomed in? Unless they are already excluded then…

Perhaps we need to play around a bit with this.

Reply:
Interesting question!
If I zoom in, the warning is only displayed for 2 points (those that allow drawing the line going out of the edge it appears). So it looks like all other points are excluded. 

The rest of the time, x axis warnings are really interesting. In my use case for instance, I needed them to debug a graph not being shown because the x axis was improperly configured.

I think we would be good if we remove 2 points from the x axis out of bounds counter. It's not going to generate false negatives except for graphs with just 2 points. While providing protection for zooming.

![image](https://github.com/user-attachments/assets/e8ae046a-f842-47fc-a478-d1b907428d91)